CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
+## 1.2.12
+* Add docker-compose files to run e2e tests with containers
```

Comment:
We do more than add docker-compose files. Do you think you may modify this?

Reply:
Updated to include information about Github workflows and automation of e2e tests